deltachat-ios/View/ContactCell.swift

```diff
@@ -401,6 +407,14 @@ class ContactCell: UITableViewCell {
         case .contact(let contactData):
             let contact = cellViewModel.dcContext.getContact(id: contactData.contactId)
             titleLabel.attributedText = cellViewModel.title.boldAt(indexes: cellViewModel.titleHighlightIndexes, fontSize: titleLabel.font.pointSize)
+
+            if let chatId = contactData.chatId {
+                let chat = cellViewModel.dcContext.getChat(chatId: chatId)
+                if chat.isProtected {
+                    subtitleLabel.isHidden = true
+                }
```

Comment:
contacts may have no chat assigned yet (the "New Chat" dialog is to create a chat in exact that case).

therefore, we need to check if a contact is verified - the same way, we show the green checkmark. this also results in email and green-checkmark being mutually exclusive.

```suggestion
            if contact.isVerified {
                subtitleLabel.isHidden = true
```

this makes things even easier and saves one db read :)

sorry to be unclear here, while writing the issue i thought it is the same condition as for the chat-titlebar (there, we know the chat exist - but the chat may be sth. else than a contact)

Reply:
Thanks a lot for pointing this out! I didn't really think about it as you were so convinced and it didn't _feel_ wrong to show the email even for contacts I don't have a verified chat with. Cause you know why should I trust people just cause we're both in a shared group?

I fixed it!